recipes-kernel/linux/linux-nezha/enable-cgroups.cfg

```diff
@@ -0,0 +1,10 @@
+CONFIG_CGROUPS=y
```

Comment:
instead of this file you can use
KERNEL_FEATURES += "features/cgroups/cgroups.cfg"